examples/DQN/atari_model.py

```diff
@@ -0,0 +1,67 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paddle.fluid as fluid
+import parl.layers as layers
+from parl.framework.model_base import Model
+from parl.utils import logger
+
+
+class AtariModel(Model):
+    def __init__(self, img_height, img_width, act_dim):
+        self.img_height = img_height
+        self.img_width = img_width
+        self.act_dim = act_dim
+
+        self.conv1 = layers.conv2d(
+            num_filters=32,
+            filter_size=[5, 5],
```

Comment:
This type of argument seems too long for writing,  could we use just `5` instead of `[5, 5]` ? 
Other frameworks support `5` as its input, which is a simpler way and good for code reading.
Could we make some change on `conv2d` in `layer_wrappers.py`?

Reply:
Paddle already support int as filter_size input, (filter_size (int|tuple|None) – The filter size. If filter_size is a tuple, it must contain two integers, (filter_size_H, filter_size_W). Otherwise, the filter will be a square.)
I will refine the implementation.